### ISSUE TITLE
MathJax v3 can’t parse MathML generated by MathJax v2

### ISSUE THREAD

**Issue by lucaswerkmeister** [mathjax/MathJax#2214]
It seems that MathJax v3 crashes when processing some MathML generated by MathJax v2. Is this to be expected? (In other words, I’m not sure if this is actually a bug or just not supported.)

Specifically, the input TeX
```tex
\mathcal M
```
is turned by MathJax v2 (via [this test page](https://gist.github.com/lucaswerkmeister/e0cb93f48f85b4dfd84292ea724bb32f), adapted from [`toMathML`](https://docs.mathjax.org/en/v2.7-latest/advanced/toMathML.html)) into the MathML
```xml
<math xmlns="http://www.w3.org/1998/Math/MathML">
  <mrow class="MJX-TeXAtom-ORD">
    <mi class="MJX-tex-caligraphic" mathvariant="script">M</mi>
  </mrow>
</math>
```
which, when pasted into the [demo section](https://www.mathjax.org/#demo) on mathjax.org, produces this preview:
> Math output error

If you enable “pause on caught exceptions” in the browser debugger, you can see that the error is

> TypeError: this.variant[t] is undefined

where `t` is `-tex-caligraphic` (sic!) and `this.variant` has no such key – the closest is <code>-tex-cal<strong>l</strong>igraphic</code>.

**Comment by dpvc** [comment]
We corrected the spelling of the calligraphic font in version 3, and that is the source of the problem here.  So I guess it is expected in that sense.  It would probably be good to have the MathML input jax in v3 translate the incorrect spelling automatically, but you could also do a global replace from `caligraphic` to `calligraphic` and that should then work in v3 as it currently stands.

**Comment by lucaswerkmeister** [comment]
Okay, thanks – after applying `.replace( /caligraphic/g, 'calligraphic' )` to the MathML, I haven’t found any other incompatibilities yet.

**Comment by dpvc** [comment]
This, and several other issues, are fixed in mathjax/MathJax-src#393.
